test/integration/functional_test_mount_test.go

```diff
@@ -280,4 +284,95 @@ func validateMountCmd(ctx context.Context, t *testing.T, profile string) { // no
 			t.Fatalf("failed to find bind address with port 46464. Mount command out: \n%v", mountText)
 		}
 	})
+
+	t.Run("verify-cleanup", func(t *testing.T) {
```

Comment:
```suggestion
	t.Run("VerifyCleanup", func(t *testing.T) {
```
We're using pascal case for test names, we still have some legacy test names using the old way